frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -274,6 +269,11 @@ export const insightLogic = kea<insightLogicType>({
             await breakpoint(100)
             actions.updateInsight({ tags: values.insight.tags?.filter((_tag) => _tag !== tag) })
         },
+        saveAsNewInsight: async () => {
+            const newInsight = await api.create('api/insight', values.insight)
+            toast(`New insight ${newInsight.name || newInsight.id} saved!`)
```

Comment:
Maybe better to do "Insight saved successfully!" as title and then a description that reads something like: "{insightName} has been saved as a new insight".